src/TabsTab.jsx

```diff
@@ -34,7 +34,10 @@ class TabsTab extends React.Component {
 }
 TabsTab.propTypes = {
 	url: React.PropTypes.string.isRequired,
-	label: React.PropTypes.string.isRequired,
+	label: React.PropTypes.oneOfType([
+		React.PropTypes.string,
+		React.PropTypes.element,
```

Comment:
yeah, we can refine the types. I'm not sure at the moment which ones other that FormatMessage and Link we would have here, but at the sametime it would be nice it was flexible to accept any time